tests/expectations/metrics/test_map_metric.py

```diff
@@ -202,6 +202,49 @@ def test_pandas_unexpected_rows_basic_result_format(dataframe_for_unexpected_row
     }
 
 
+def test_pandas_unexpected_rows_summary_result_format(dataframe_for_unexpected_rows):
```

Comment:
Maybe we also need a test with `include_unexpected_rows: False` and make sure that metric isn't computed?

Reply:
great catch. Thank you